application/invites/invites_view.py

```diff
@@ -0,0 +1,32 @@
+"""Handle requests on the team resource."""
+from flask import abort
+from flask_apispec import use_kwargs, marshal_with
+from flask_apispec.views import MethodResource
+from flask_jwt_extended import get_jwt_identity, jwt_required
+
+from application.teams.team import Team
+from application.teams.team_schema import TEAM_SCHEMA
+from application.users.user import User
+from .invites_validator import accept_invite_args
+
+
+class AcceptInvite(MethodResource):
+    """Controller for invite links."""
+
+    @jwt_required
+    @use_kwargs(accept_invite_args())
+    @marshal_with(TEAM_SCHEMA)
+    def post(self, **params):
+        """Logic for adding a single team member"""
+        print('JWT: ' + str(get_jwt_identity()))
+        user = User.find_by(id_=get_jwt_identity())
+        invite_link = params['join_key']
+        team = Team.find_by(join_key=invite_link, force=False)

```

Comment:
why the renaming?
just use 
`team = Team.find_by(join_key=params['join_key'], force=False)`

Reply:
What does force do?

Reply:
force=True causes the function to fail and raise an exception if the team is not found, whereas force=False will let it return none.
I am checking for team=None on the next line and returning a 404 if the team does not exist, so an exception is not needed at this point.

Reply:
Imo `except NotFound` is more expressive and you don't have the extra force variable which apparently can be misunderstood.